components/Testimonial.js

```diff
@@ -1,35 +1,128 @@
-import Paragraph from "./typography/Paragraph";
-
-export default function Testimonial({
-  className = '',
-  text,
-  authorName,
-  authorDescription,
-  authorAvatar,
-}) {
+import { FaArrowLeft, FaArrowRight } from 'react-icons/fa'
+
+const testimonials = [
+  {
+    imgUrl: '/img/testimonials/matt-mclarty.jpg',
+    name: 'Matt McLarty',
+    salutation: 'Global Leader of API Strategy at MuleSoft',
+    text: 'Microservices underline the need for event-based communication in distributed architectures. AsyncAPI brings the richness of the REST API ecosystem to asynchronous APIs.',
+  },
+  {
+    imgUrl: '/img/testimonials/bill-doerrfeld.jpg',
+    name: 'Bill Doerrfeld',
+    salutation: 'Editor in Chief at Nordic APIs',
+    text: 'Microservices underline the need for event-based communication in distributed architectures. AsyncAPI brings the richness of the REST API ecosystem to asynchronous APIs.',
+  },
+  {
+    imgUrl: '/img/testimonials/jonathan-schabowsky.jpg',
+    name: 'Jonathan Schabowsky',
+    salutation: 'Sr. Architect, Office of the CTO at Solace',
+    text: "Developers need to be able to quickly and consistently create event-driven applications that provide business value and react to customer needs in realtime. I can't count how many times I've heard developers ask for OpenAPI/Swagger style tools for the asynchronous and event driven world, and that is exactly what the AsyncAPI initiative is making a reality.",
+  },
+  {
+    imgUrl: '/img/testimonials/eric-horesnyi.jpg',
+    name: 'Eric Horesnyi',
+    salutation: 'CEO at Streamdata.io',
+    text: 'We’ve been focusing on event-driven APIs since 2014 and thank the AsyncAPI contributors everyday for driving the community towards common standards.',
+  },
+];
+import React, { useState } from 'react';
+
+const Testimonial = () => {
+  const len = testimonials.length;
+  const [activeIndex, setActive] = useState(0);
+
+  const getStyle = (idx) => {
```

Comment:
Kindly mention the logic you used to add the styles inside testimonials cards in the form of comments.